quantum/quantum_keycodes.h

```diff
@@ -681,6 +688,13 @@ enum quantum_keycodes {
     #define X(n) (QK_UNICODE_MAP | (n))
 #endif
 
+#define UC_M_OS UNICODE_MODE_OSX
+#define UC_M_LN UNICODE_MODE_LNX
+#define UC_M_BD UNICODE_MODE_BSD
```

Comment:
Maybe `UC_M_BS`? Because **OS**X, **LN**X, **BS**D, **WI**N.

Reply:
Yeah, I considered that, but it's too much BS, if you ask me.

Reply:
```suggestion
#define UC_M_BS UNICODE_MODE_BSD
```

Reply:
> but it's too much BS, if you ask me.

Oof, where's the 👌 reaction when you need it
(this is why I use GitLab)

Reply:
:)

I did actually change it, but .... yeah, I'm waiting for somebody to make a joke about this elsewhere.

Also, for what BSD stands for, BS makes more sense.

Reply:
At the risk of too many cooks, why not `UC_MBSD`, etc.? Or `UCM_BSD`?

Reply:
Well, honestly, BSD isn't even supported at the moment. It's a placeholder for when somebody with BSD wants to help out.

Reply:
I meant the short keycode format. I don't see why the M needs an underscore on each side.

Reply:
Well, we can't use `UC_LNX` and the like, because that's already defined and in use.  So need a code that works.  `UCM_*` didn't look "good" to me.  What I was using was worse, actually.  

So, really, aesthetics.

Reply:
¯\\\_(ツ)_/¯

Reply:
It's not used, and it works, and it conforms to how the rgb codes look.  So ...

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L584-L592